docs/pages/docs/guides/auth-and-access-control.mdx

```diff
@@ -0,0 +1,532 @@
+
+The result of the query is stored in `context.session`, and available to almost every function and hook in Keystone – including access control.
+
+In this initialisation phase, you'll want to load any data you'll need to work out what the user should be able to do (and what they should not be able to do).
+
+If no session cookie is present, or no matching item can be found, `context.session` will be `undefined`.
+
+### Add a sessionData query
+
+In our example above, we probably want to know the current user's ID and the value of the `isAdmin` checkbox. The auth package makes this simple by providing a `sessionData` option, which should contain the fields to query when a session is found.
+
+You configure it like this:
+
+```ts
+const { withAuth } = createAuth({
+  // ...
+  sessionData: 'name isAdmin',
+});
+```
+
+Think of this like the field selection in a GraphQL query. You can load any fields you need to have at hand when writing Access Control methods, including virtual fields and relationships.
+
+The equivalent GraphQL query would look like this:
+
+```graphql
+query {
+  person(where: { id: $session.itemId }) {
+    name
+    isAdmin
+  }
+}
+```
+
+## Adding Access Control
+
+Now that we have information about the current user, lets put it to good use and add some security to our API with **Access Control**.
+
+The way to think about access control is –
+
+- With **Session Data**, you know who the user is, and what they should have access to (whether that's permissions granted by a role, or items related to them)
+- For each **List** you can restrict which operations they can perform, filter the items they have access to, and control which fields they can read and update.
+
+!> In other words: **Step 1** load data about the current user; **Step 2** use that data to limit what they can do.
+
+You define access control for each List (and field) individually; the rules you specify will always be run, even when the list being queried or mutated through a relationship.
+
+### Posts Example
+
+Let's say we have a list of Blog Posts, which have a published status. We want Admin users to see all blog posts and be able to crete and update them, but anonymous (public) users can only see published posts and can't write to the API. Our Post list might look like this:
+
+```ts
+const Post = list({
+  fields: {
+    title: text(),
+    isPublished: checkbox(),
+    publishDate: timestamp(),
+    author: relationship({ ref: 'Person' }),
+    // more content fields would go here
+  },
+});
+```
+
+And the Session Data we set up above would look like this:
+
+```ts
+type Session = {
+  data: {
+    id: string;
+    isAdmin: boolean;
+  }
+}
+```
+
+We can now set up operational access control to restrict the **create**, **update** and **delete** operations to authenticated users with the `isAdmin` checkbox set:
```

Comment:
we embolden `filter` access control below, but not here, this seems inconsistent.

Reply:
Ha thanks, I changed conventions halfway through and knew I'd miss one 🙂 

Fixed